.eslintrc.json

```diff
@@ -1,17 +1,14 @@
 {
```

Comment:
Do we really need a linter for the SSCCEs?

Reply:
The already was one so I took the liberty of aligning it with the main repo but at the same time I don't want end-users to be annoyed by it.
I think I'll keep it on our code, but lintignore the sscce folder